src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker/mediapicker.controller.js

```diff
@@ -65,7 +67,7 @@ angular.module('umbraco').controller("Umbraco.PropertyEditors.MediaPickerControl
 
            $scope.mediaPickerOverlay = {
                view: "mediapicker",
-               title: "Select media",
+               title: localizationService.localize("defaultdialogs_selectMedia"),

```

Comment:
`title` can be omitted here since it fallback to this value in `umb-overlay` as well, but it might be more clear to add it here too.